device.yml

```diff
@@ -325,18 +325,21 @@ registers:
     access: Write
     maskType: Channel3RangeConfig
     description: Selects the flow range for the channel 3.
-  Reserved0: &reserved
+  TemperatureValue: 
     address: 92
     type: U8
     access: Read
-    description: Reserved for future use.
-    visibility: private
-  Reserved1:
-    <<: *reserved
+    description: Temperature sensor reading value.
+  EnableTemperatureCalibration:
     address: 93
-  Reserved2:
-    <<: *reserved
+    type: U8
+    access: Read
+    description: Enable flow adjustment based on the temperature calibration.
+  UserTemperatureCalibration:
     address: 94
+    type: U8
+    access: Read
+    description: User temperature calibration value.
```

Comment:
If appropriate, it would be helpful to have a more explicit description here.

Reply:
Sure, will try to make it more clear.